css/src/components/tooltip.css

```diff
@@ -11,7 +11,151 @@
   transition-timing-function: cubic-bezier(0, 0, 0.38, 0.9);
 }
 
-.Tooltip-text {
+.Mds-Tooltip-text {
   word-break: break-word;
   hyphens: auto;
 }
+
+@keyframes Mds-tooltip-open-top {
+  from {
+    margin-top: 0px;
+    opacity: 0.5;
+  }
+
+  to {
+    margin-top: -8px;
+    opacity: 1;
+  }
+}
+
+@keyframes Mds-tooltip-close-top {
+  from {
+    margin-top: -8px;
+    opacity: 0.5;
+  }
+
+  to {
+    margin-top: 0px;
+    opacity: 0;
+  }
+}
+
+.Mds-Tooltip-animation-open-top {
+  animation: Mds-tooltip-open-top 120ms cubic-bezier(0, 0, 0.3, 1);
+  animation-fill-mode: forwards;
+}
+
+.Mds-Tooltip-animation-close-top {
+  animation: Mds-tooltip-close-top 120ms cubic-bezier(0.4, 0.14, 1, 1);
+}
+
+@-webkit-keyframes Mds-tooltip-open-bottom {
+  from {
+    margin-top: 0px;
+    opacity: 0.5;
+  }
+
+  to {
+    margin-top: 8px;
```

Comment:
spacing between trigger and tooltip should be 4px. Please refer this https://design.innovaccer.com/0ee0e5c39/v/0/p/1014fc-tooltip/t/271a0a